### PR TITLE
added model and tests

### PR DESCRIPTION
**Pull request by ramesht007**
## Description
Created a local language model for Syfertext_0.1.0 addressing issue #142

### CODE REVIEW

File: tests/utils.py [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+    vocab_size = len(vocabulary)
+    # creating index for vocab
+    indexes = list(range(vocab_size))
+    # create hash2row empty dict
+    hash2row = {}
+    # create hash2row dict
+    for i in range(vocab_size):
+        index = random.choice(indexes)
+        indexes.remove(index)
+        hash2row[hash_string(vocabulary[index])] = index
```
Comment (AlanAboudib) [inline tests/utils.py:66]: no need for randomization here. you can just do:

```
for i, word in vocabulary:
   hash2row[hash_string[word]] = i
```

File: tests/utils.py [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+import random
+import numpy as np
+
+
+def get_test_language_model():
+    nlp = syfertext.create(model_name="syfertext_sentiment")
+    # Create the tokenizer
+    tokenizer = Tokenizer()
+
+    vocabulary = [
```
Comment (AlanAboudib) [inline tests/utils.py:18]: The list of words should also include all the words that are used in the tests. I suggest actually that you take the list of hashes from 'syfertext_en_lang_web_lg'. 

You can create a file containing this list of hashes, you put it in tests/vocab_hashes.py. This way we are sure we have all the necessary words.

Testing using this locally created language model will be changed again once @Nilanshrajput integrated the grid nodes.

File: tests/utils.py (hunk 1 above)
Comment (ramesht007) [reply tests/utils.py:66]: ok

File: tests/utils.py (hunk 2 above)
Comment (ramesht007) [reply tests/utils.py:18]: ok I will make the changes

**Comment by AlanAboudib** [comment]
Done by @ramesht007 in #163

**Pull request closed without merging**